steps/step-01.txt

```diff
@@ -0,0 +1,26 @@
+/** 
+Let's start creating our Tic Tac Toe game!
+
+Step #1
+
+1. Create file index.html
+2. Open file and put following text inside:
+this will be our main document.
+
+  <html>
+  <body>
+    <div class="board-frame">
+      <div>
+        <button>0</button>
```

Comment:
This I think should be O (capital o) instead of 0 (zero) as oppose to X.
Also in this step:
<button onclick="setField(0)">0</button>
there are two 0s so changing to O clear the confusion of which one should be edited.

Reply:
Oh, there is probably a slight confusion here: this particular "0" is zero, cause it is a number on the button. We have numbers on buttons to make indexing and search for cells more straightforward.